### ISSUE TITLE
The watch stays in "alarm" state even after answering a call on the phone

### ISSUE THREAD

**Issue by vaspa** [ossw/ossw-android#24]
This problem appears on 4.4.2 stock (Galaxy Note 2)

**Comment by althink** [comment]
It's quite hard to fix. On my girlfriend's htc phone with 4.4.2 when you answer the call this incoming call notification is still present with the same id (2) and the only difference is the notification text (which is language dependent) and notification flags, before answering the call it's 0x82, after answering it's 0x2.... but this flag (0x80) is deprecated since API 16 according to doc :/

**Comment by vaspa** [comment]
Yes, on my side the flag is always 2, that is why I started thinking about a separate module for managing incoming calls on Android 4.3-4.4.*.

**Comment by mparol** [comment]
Maybe instead of relying on notifications it has to be done using intents, like in http://stackoverflow.com/questions/15563921/detecting-an-incoming-call-coming-to-an-android-device

**Comment by vaspa** [comment]
Yes, also have a look at #25. Its another way to treat phone state changes, but apparently not working the same way in all Android versions.
I was planning to implement and test this for KitKat only. Will spend some time on it in the following days.
